Code/Core/Process/Process.cpp

```diff
@@ -435,14 +458,6 @@ int Process::WaitForExit()
         VERIFY( CloseHandle( GetProcessInfo().hProcess ) );
         VERIFY( CloseHandle( GetProcessInfo().hThread ) );
 
-        if ( !m_SharingHandles && m_RedirectHandles )
-        {
```

Comment:
We still need to call CloseHandle on the duplicate handles, or this will result in handle leaks.